afdko/Tools/Programs/public/lib/api/supportenvironment.h

```diff
@@ -285,7 +283,7 @@ that changes to environment.h be upward-compatible, as described above.
  */
 
 #ifndef PROTOTYPES
-#define PROTOTYPES	(OS!=os_sun)
+#define PROTOTYPES	(OS)
```

Comment:
That's not how this is meant to be used -- see the comment block above this line for more information about it, but it doesn't seem to be used at all, so I think you should remove this and the comment block above it.